### PR TITLE
refactor: remove traceroute feature

### PR DESCRIPTION
**Pull request by RolandSherwin**

### CODE REVIEW

File: sn_client/src/connections/messaging.rs [hunk 1]
```diff
@@ -234,9 +217,6 @@ impl Session {
         #[allow(unused_mut)]
         let mut wire_msg = WireMsg::new_msg(msg_id, payload, kind, dst);
```
Comment (b-zee) [inline sn_client/src/connections/messaging.rs:219]:
```suggestion
        let wire_msg = WireMsg::new_msg(msg_id, payload, kind, dst);
```

**Review by b-zee — CHANGES_REQUESTED** [review]
Good stuff, unfortunate that we couldn't use this very much as we thought we might...

File: sn_node/src/node/data/records.rs [hunk 2]
```diff
@@ -387,14 +340,6 @@ impl MyNode {
         #[allow(unused_mut)]
```
Comment (b-zee) [inline sn_node/src/node/data/records.rs:340]: Same applies here.

Comment (RolandSherwin) [reply sn_node/src/node/data/records.rs:340]: The other feature gated code block which calls `set_payload_debug()` requires `wire_msg` to be mutable..

Comment (b-zee) [reply sn_node/src/node/data/records.rs:340]: Aah, sorry, that was meant for another piece in the same file, don't know what happened. Solved!

**Review by b-zee — APPROVED** [review]

**Review by davidrusu — APPROVED** [review]

**Pull request merged**